app/src/components/NextButton.tsx

```diff
@@ -0,0 +1,7 @@
+import MainButton from './MainButton'
```

Comment:
Where's `MainButton`? :roll_eyes: 
If you need that for this PR, it should include those commits too.